spec/aruba/api/commands_spec.rb

```diff
@@ -26,6 +28,15 @@
         @aruba.write_file(@file_name, "Hello\nWorld!")
         @aruba.pipe_in_file(@file_name)
         @aruba.close_input
+
+        @aruba.last_command_started.stop
+        last_command_output = @aruba.last_command_started.output
+
+        # Convert \r\n to \n, if present in the output
+        if last_command_output.include?("\r\n")
+          allow(@aruba.last_command_started).to receive(:output).and_return(last_command_output.gsub("\r\n", "\n"))
```

Comment:
Maybe better to either:
* make `have_output` behave properly in this case,
* provide a special verision of `have_output` that ignores changes to line endings.
* or change the expectation below depending on whethere we're on Windows.

I think I prefer the third option for now.